### PR TITLE
New random in ifs section following Kate's great suggestion!

### PR DESCRIPTION
**Pull request by bhoffman0**

### CODE REVIEW

File: _sources/Unit3-If-Statements/topic-3-3-if-else.rst [hunk 1]
```diff
@@ -417,6 +417,149 @@ You can use curly braces (``{}``) to enclose a nested if and have the else claus
 In fact many experienced Java programmers `always` use curly braces, even when
 they are not technically required to avoid this kind of confusion.
 
+Math.random() in if Statements
+-------------------------------
+
+The ``Math.random()`` method returns a random number between 0.0 and 1.0.  You can use this method with ``if`` statements to simulate a coin flip or an event occuring a certain percentage of the time. For example, if you want to simulate a coin flip, you can check if the random number is less than 0.5 (halfway between 0 and .1) to simulate a 50% chance of heads or tails:
```
Comment (gigamonkey) [inline _sources/Unit3-If-Statements/topic-3-3-if-else.rst:423]:
```suggestion
The ``Math.random()`` method returns a random number between 0.0 and 1.0.  You can use this method with ``if`` statements to simulate a coin flip or an event occuring a certain percentage of the time. For example, if you want to simulate a coin flip, you can check if the random number is less than 0.5 (halfway between 0 and 1) to simulate a 50% chance of heads or tails:
```

**Review by gigamonkey — APPROVED** [review]
One small typo.

File: _sources/Unit3-If-Statements/topic-3-3-if-else.rst [hunk 2]
```diff
@@ -417,6 +417,149 @@ You can use curly braces (``{}``) to enclose a nested if and have the else claus
...
+        
+         if (Math.random() < 0.25) { System.out.println("Rain"); }
+       
+      + Correct! This code will print "Rain" 25% of the time.
+
+   -  .. code-block:: java
+        
+         if (Math.random() > 0.75) { System.out.println("Rain"); }
+
+      - Correct. This code will print "Rain" 25% (.1 - .75) of the time.
```
Comment (bhoffman0) [inline _sources/Unit3-If-Statements/topic-3-3-if-else.rst:470]:
```suggestion
      + Correct. This code will print "Rain" 25% (.1 - .75) of the time.
```

File: _sources/Unit3-If-Statements/topic-3-3-if-else.rst [hunk 3]
```diff
@@ -417,6 +417,149 @@ You can use curly braces (``{}``) to enclose a nested if and have the else claus
...
+        
+         if (Math.random() < 0.25) { System.out.println("Rain"); }
+       
+      + Correct! This code will print "Rain" 25% of the time.
+
+   -  .. code-block:: java
+        
+         if (Math.random() > 0.75) { System.out.println("Rain"); }
+
+      + Correct. This code will print "Rain" 25% (.1 - .75) of the time.
```
Comment (bhoffman0) [inline _sources/Unit3-If-Statements/topic-3-3-if-else.rst:470]:
```suggestion
      + Correct. This code will print "Rain" 25% (1 - .75) of the time.
```

**Pull request merged**